contracts/core/EntryPoint.sol

```diff
@@ -24,8 +23,13 @@ contract EntryPoint is IEntryPoint, StakeManager {
 
     SenderCreator private immutable senderCreator = new SenderCreator();
 
+    /**
+     * for simulation purposes, account should accept signature
+     * failure if this is the given aggregator.
```

Comment:
Maybe add that EntryPoint guarantees that this UserOperation will revert without side effects if called on-chain.